src/components/configure/keymapToolbar/KeymapToolbar.tsx

```diff
@@ -321,7 +350,56 @@ export default class KeymapMenu extends React.Component<
           productId={productId}
           productName={productName}
         />
+        <ConfirmDialog
+          open={this.state.openConfirmDialog}
+          onNoClick={this.onClickConfirmDialogNo.bind(this)}
+          onYesClick={this.onClickConfirmDialogYes.bind(this)}
+        />
       </React.Fragment>
     );
   }
 }
+
+type IConfirmDialogProps = {
+  open: boolean;
+  onYesClick: () => void;
+  onNoClick: () => void;
+};
+
+function ConfirmDialog(props: IConfirmDialogProps) {
+  return (
+    <Dialog
+      open={props.open}
+      aria-labelledby="confirm-dialog-title"
+      aria-describedby="confirm-dialog-description"
+    >
+      <DialogTitle id="confirm-dialog-title">{'Reset Keymap'}</DialogTitle>
+      <DialogContent>
+        <DialogContentText id="confirm-dialog-description" color="secondary">
+          Are you sure to reset keymap?
+          <br />
+          (Flashed immediately, save current keymap before resetting.)
```

Comment:
I think, it might be better to use "future tense" like as follow:

`
Current keymap will be discarded and an initial keymap will be applied immediately. If you want to reuse current keymap, please save it before resetting.
`

Reply:
@yoichiro LGTM for the logic. It might be rethink the alert text, I think.

Reply:
@adamrocker Thank you for the comment. I updated the dialog title and the sentence at f1b351c.